### ISSUE TITLE
Please Invite Me to #OpenLibrary Slack

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#686]
# Slack Invites for Active Contributors
Most of our day-to-day chat happens on #openlibrary-g -- a public community channel on our archive.org slack. We're happy to send invites to **Active Contributors** who have worked through setting up Docker and or have demonstrated commitment by discussing or working on issues. If you are an active contributor and would like to be apart of our slack channel, please let us know here. If your github email is not public, please send an email to mek@archive.org so we can add you.

## Requirements
**What is an Active Contributor?** An active contributor has done one or more of the following things:
- You've setup docker (or you've hit a problem with docker and need help)
- You've attended one of our community calls
- You've created an issue or commented meaningfully on an existing issue
- You've submitted a meaningful PR to address an issue

## Why Are There Requirements?
As you can see in the comments below, we receive a large volume of requests from community members to join our slack channel. A good portion of the folks we invite don't end up participating making it hard for new members to figure out who the active members are. As a result, we have decided to reserve slack as a place where trusted active contributors can meet, discuss issues, and unblock each other.

## But I Need Help!
We've got you covered -- If you're just getting started, we have a [gitter channel](https://gitter.im/theopenlibrary/Lobby) (https://gitter.im/theopenlibrary/Lobby) where you can ask troubleshooting questions and receive help choosing a first issue.

**Comment by salman-bhai** [comment]
@mekarpeles I feel it would be better if we put this maybe in the README rather than over here as it's too many tabs for a contributor to traverse through to find this! We already have put out Gitter Channel link on the README, so maybe we could have that first-time contributors can ask questions on the Gitter channel and then get added to the Slack channel. As in this entire thing could be one process!

What do you think about it?

**Comment by himanshu1603** [comment]
How can I join the community?

**Comment by mekarpeles** [comment]
Hi @himanshu1603! We'd love you to tell us more about what you're interested in and we can help you contribute to the Open Library project. You can reply on this thread, or you can chat with us on https://gitter.im/theopenlibrary/Lobby.

**Comment by seabelis** [comment]
I'd be interested, though I'm not sure how much I can contribute as a non-developer.

**Comment by mekarpeles** [comment]
@seabelis, if you send me an email <mek@archive.org> we'd be happy to invite you to our slack channel. Designers and librarians are still developers and we're happy to receive input and contributions from everyone! Thanks for lending your voice :) and amazing job with the logos, this was such a big help.

**Comment by LoganSimonsen** [comment]
Hello Everyone, I am a Web Developer student at DevMountain Bootcamp and I am building a React application for a project assignment that uses the Open Library API to search books and display results. I am thinking that  in the future I could potentially convert this to React Native as a mobile application. Here is my project (still work in progress): https://github.com/LoganSimonsen/no-db-project 

Do you know if anyone has built a mobile application already for Open Library?

**Comment by mekarpeles** [comment]
@LoganSimonsen @rchrd2 started working on a proof of concept for a progressive mobile web app -- you may want to comment and tag him on this project https://github.com/ArchiveLabs/openlibrary-lite

**Comment by LoganSimonsen** [comment]
I've hosted the app I built if anyone would like to check it out.

 openlibrarysearch.now.sh

**Comment by manan-goel-1999** [comment]
I'm new to opensource and would like to start contributing. I was wondering if I could be added to the slack channel

**Comment by rachelmoore** [comment]
Hi there! I am a JS dev in SF and have volunteered with Internet Archive in the past (just sorting media). Interested in helping out with frontend stuff on Open Library and would like to join the Slack channel.

**Comment by nirabpudasaini** [comment]
I am a mobile developer with some js experience, i am looking to contribute to a project where i will also have a chance to learn. I also love the work openlibrary is doing. I am hoping to set up and look through the first issues in coming days.

**Comment by mekarpeles** [comment]
@nirabpudasaini, @rachelmoore, @manan-goel-1999 -- you should receive invited by Tuesday (US Pacific Time). Thank you for your interest, we're looking forward to welcoming you to our community on slack :)

**Comment by FrackeR011** [comment]
Hi ! I am new to open source world but genuinly interested in contributing. I am learning javascript an have familiarity with html and css.

**Comment by FrackeR011** [comment]
Hi ! I am new to open source world but genuinly interested in contributing. I am learning javascript an have familiarity with html and css.

**Comment by mekarpeles** [comment]
@FrackeR011 we need your email to do so, please email mek@archive.org so we can add you

**Comment by tabshaikh** [comment]
@mekarpeles maybe we should think of redirecting to a readme as said by @salman-bhai

**Comment by aman-tugnawat** [comment]
Would love to be a part of the community mailto::aman.tugnawat@gmail.com. Please send an invite https://google.com/search?q=Aman+Tugnawat

#wayback #beta #backrub #sergey #larry #sundar sundar@google.com

**Comment by debck** [comment]
Hi everyone , I would also like to contribute to the organistaion. Please send me an invite to slack channel.
Thnx.

**Comment by w3bh4ck** [comment]
Hi, I just got to find out about archive.org today and It was really helpful for what I needed to do at that moment. Although I may not be able to make a reasonable donation, I will like to help out in the area of development.  I  am a full-stack JS developer and can comfortably work with PHP and Python also. If you need extra hands as volunteers, I will be glad to help out. Email: amadiluckysampson@gmail.com

**Comment by viccro** [comment]
Hi! I'm interested in contributing; please add me?

**Comment by mekarpeles** [comment]
@w3bh4ck and @viccro are our last invites for the time being!

We have a large number of guests in our openlibrary slack channel who are are inactive and because of this, future invitations will require at least 1 contribution to Open Library. :)

Contributions might include code / PRs or code reviews, design, improvements to our documentation / wiki organization, fixes to book catalog data, adding new issues, QA testing.

**Comment by mekarpeles** [comment]
p.s. @viccro we'll need your email to add you. Feel free to ping me at mek@archive.org

**Comment by robla** [comment]
Hi @mekarpeles , I'd love to continue some of the conversations I had with all y'all at the [DWebSummit](https://decentralizedweb.net/).  I'm particularly interested in exploring more about podcast metadata and also bookmarking.

**Comment by Alakhator** [comment]
I am new to open source. So please add me to your slack channel. I have sent you an email as well

**Comment by salman-bhai** [comment]
Hi @Alakhator welcome to Open Library. Here are a [couple of issues](https://github.com/internetarchive/openlibrary/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) that you can resolve before getting invited to Open Library.

**Comment by 3srliton** [comment]
3srliton@gmail.com

**Comment by tabshaikh** [comment]
@3srliton  Before getting involved you must solve at least one issue. This is done so that only motivated individuals are invited to the slack channel and it avoids overcrowding of the slack channels too

**Comment by salihkeles** [comment]
Hi how can i contribute as a developer and join the community

**Comment by mekarpeles** [comment]
Howdy @salihkeles -- can you tell us more about what type of improvements you're interested in helping with? What's your goal :)
 
A great place to start is checking our open issues
github.com/internetarchive/openlibrary/issues

**Comment by thebuizelclub** [comment]
Hey I'm interested in helping with the backend in possibly squishing some bugs. I'm totally new to open source development and still a bit new to python but I still think I could be able to be some sort of help.

**Comment by mekarpeles** [comment]
Hi @Ayshhh, thanks for helping out :)
A first great step is trying to get docker running -- if you're open to trying, we'd love to know if you experience any trouble!

**Comment by aditya81070** [comment]
Hello, @mekarpeles   I want to contribute to Internet Archive open source projects. I know javascript and react. How can I join slack and how can I get start to contribute?

**Comment by Ayshhh** [comment]
How can I help you in contributing any project from my side?

**Comment by salman-bhai** [comment]
Hi we're active on our Gitter channel on getting any first timer to get started with Open Library.

**Comment by salman-bhai** [comment]
Please do take a look at our especially the one with the tag "Hacktoberfest"

**Comment by aditya81070** [comment]
@salman-bhai where I can find gitter link?

**Comment by mekarpeles** [comment]
@aditya81070 here you go!
https://gitter.im/theopenlibrary/Lobby

**Comment by KhalidRmb** [comment]
Hi! I'm new here and have made 2 PRs so far, and I would love to keep up the contributions. May I get an invite to the slack channel? @mekarpeles @salman-bhai

**Comment by mekarpeles** [comment]
@khalidrmb absolutely :)

**Comment by Alakhator** [comment]
I have made 1 pull request and would continue to do so. Please send me an
invite to slack channel.

Alakh Singh Sethi
COE 8

**Comment by himanshuc3** [comment]
Hi. I would like to join openlibrary@slack. My id is himichhabra14@gmail.com.

**Comment by usmanmuhd** [comment]
Hi I would like to join slack. My email-id is `me@usmanmuhd.com`

**Comment by yogeshjdeshmukh** [comment]
Hi, I just came to know about slack community of open library. I have contributed 1 PR so far and would like to contribute more. I would like to join the slack community. My email id is yogeshjdeshmukh1999@gmail.com

**Comment by Whales1992** [comment]
interested.  kola.victor@yahoo.com

**Comment by yagyanshbhatia** [comment]
hi @mekarpeles !
I think my beliefs resonate with those of archive.org and I'm very excited to contribute! Please add me to the slack group :smile:

**Comment by scottrlarson** [comment]
I would like to add a table of contents to “"An Etymological Dictionary of Modern English“ epub format at 
https://archive.org/details/etymologicaldict00weekuoft. does the archive accept these kinds of contributions?

**Comment by adenrulz** [comment]
Hi, I'd like to join the slack channel. I'm a jr front end developer. I could help for the design.

mihrice@gmail.com

Universal access to all knowledge

**Comment by mekarpeles** [comment]
@scottrlarson unfortunately you'd have to add the table of contents to the corresponding Open Library page -- archive.org pages are not community/wiki editable, sorry! Happy to answer further if you open a separate issue on https://github.com/internetarchive/openlibrary/issues :)

**Comment by mekarpeles** [comment]
Hi y'all -- we have a large handful of people in our slack channel who are not active contributors and so we're reserving slack as a place where trusted active contributors can meet, discuss, and unblock each other.

We're happy to send invites to folks who have worked through get Docker up and running and or have worked on an issue. We also have a gitter channel (https://gitter.im/theopenlibrary/Lobby) if you have questions about choosing your first issue or if you experience problems with getting docker/openlibrary setup.

cc: @ayushshivani, @adenrulz, @Whales1992

**Comment by gadgrandez** [comment]
Hello, I would like to contribute to the development of OpenLibrary, I would like to start with the translations of the site into Spanish and later with programming tasks, if someone tells me what to do I will be happy to help :smile:

**Comment by pprathameshmore** [comment]
Hello there,
I am Prathamesh More. I interested to contribute open source community to be part of an open source development.
moreprathameshmore@hotmail.com

**Comment by tabshaikh** [comment]
@gadgrandez You could began contributing to internationalization by setting up open library on your local system and then trying out the steps in this readme to began with spanish translations https://github.com/internetarchive/openlibrary/i18n/README.md

**Comment by gadgrandez** [comment]
@tabshaikh Thanks, I'll get into it.

**Comment by theaayush8999** [comment]
I would like to contribute send me the invite !!
email : this.aayush@gmail.com

**Comment by shreyansh23** [comment]
@mekarpeles 
I'd love to join the slack channel. I have sent you an email at mek@archive.org for the same.

**Comment by Ifeanyijasper** [comment]
Hey I will like to contribute. This is my email ifeanyijasper11@gmail.com

**Comment by rickhohler** [comment]
I would like to contribute. Email is rick.hohler@gmail.com

**Comment by mekarpeles** [comment]
Howdy @rickhohler + @Ifeanyijasper, thanks for joining the community! Excited to work on issues together. Have you been successful in getting the Open Library dev environment setup using docker?

We're currently trying to keep our community slack focused for active contributors -- if this is you, please let us know some of the issues you've been working on and we'll be glad to send you a slack invite!@ 
https://github.com/internetarchive/openlibrary/issues/686#issuecomment-450934598

**Comment by Ifeanyijasper** [comment]
I have installed Open Library dev environment setup using docker successfully and I want to work on this issue https://github.com/internetarchive/openlibrary/issues/1986 but I don't know how to go about it. That is to reproduce the error on my local machine and getting the source code.

**Comment by mercy254** [comment]
Hi
I would like to join your slack channel my email is **muthonimercy054@gmail.com**

**Comment by mekarpeles** [comment]
Hi @mercy254 slack access is currently only for active contributors to the open library project :)

**Comment by schippas** [comment]
Hello, I am interested in contributing to this project. Are there still community callouts, and if so how do I access them? I was thinking of working on #2085, #2086, and #2087 to start out and get familiar with the codebase.

**Comment by illseption** [comment]
Hi! I'm a newbie and I'd like to contribute to the openlibrary project. I've installed the docker images and was looking for next steps. Can I be added to the slack channel?

**Comment by Ishaan28malik** [comment]
Hey 
I would like to join the slack channel and contribute @mekarpeles

**Comment by darkbluewalrus** [comment]
Hello! I would also like to join the slack channel and contribute @mekarpeles. Thanks!

**Comment by imnotjames** [comment]
Hi there.  I'm interested in helping with the project and would be interested in listening to the discussions that happen around the OpenLibrary project.

**Comment by Mfhodges** [comment]
Hello!
I would like to join the slack channel and contribute @mekarpeles
Thank you!

**Comment by cdrini** [comment]
@mekarpeles Could you add @nilshah98 to the slack channel? They just completed their first PR :) #2861

**Comment by nilshah98** [comment]
Thanks a lot @cdrini .
I look ahead to get started and contribute to this community.

Specifically I am targeting GSOC 2020, and would like to know if there are any updates on the same.
Cheers, and thanks for having me again!

**Comment by Ma-Sam** [comment]
Hello everyone, I have C ++ experience and would like to contribute to this community.

**Comment by user404d** [comment]
Hello~ Looking for an invite. I have experience with docker, python, and js (though I have not done much frontend work recently). Would love to see how I can contribute to the project.

**Comment by aashimgarg** [comment]
Hello. I am looking for an invite in the slack channel. I recently made in 2 commits in the internet/openlibrary repository. I have experience in html,css,bootstrap,javascript frameworks. 
would love to contribute to this project.

**Comment by yash-gupta2000** [comment]
I'm GSOC 2020 aspirant and am having problem in installing docker images. Error shown is
Sending build context to Docker daemon  14.85kB
Step 1/12 : FROM olbase
pull access denied for olbase, repository does not exist or may require 'docker login': denied: requested access to the resource is denied

I want to contribute to openlibrary but am stuck at this for past 3 hours. Help?

**Comment by yash-gupta2000** [comment]
Hello, I'm GSOC 2020 aspirant looking to contribute to open library.
I made PR #3083 which is showing tick along with Travis CI build passed. I'm looking for invite in slack channel. I have hands-on experience in C/C++ , HTML, Bootstrap and JS and willing to contribute to this project.

**Comment by manav014** [comment]
Hello, I want to contribute to the open library for GSoC 2020. I have made a PR #3078 and have compiled the whole software with docker and even has set up the development environment for the same.  I would be glad if you may please allow me to join the slack channel so that I could make my journey of GSoC with open library the best.

**Comment by mightysood** [comment]
Hi @cdrini @mekarpeles 
I have set up the project locally and recently made a PR #3205 and now working on more issues. It would be great if I could get an invite for the slack channel as it would help me contribute better.

**Comment by mackinder1915** [comment]
test

**Comment by jamesachamp** [comment]
Please add me to your slack channel.  I've set up my local development environment, created PR [#3754](https://github.com/internetarchive/openlibrary/pull/3754), and intend to make additional contributions.  Thanks!

**Comment by NanaYawBotar** [comment]
Please add me to your slack channel

**Comment by codetochangeMinds** [comment]
I want to contribute to open library project and looking to be a part of this community. I have done a handful of projects using python and javascript. Till now I have gone through the readme, wiki page and done with the docker installation. I am interested to contribute and would like to join slack channel.

**Comment by Yashs911** [comment]
@mekarpeles Please add me to your slack channel.

**Comment by BrittanyBunk** [comment]
I'd like to be invited to slack

**Comment by Zorlin** [comment]
@mekarpeles I'd like to be invited to the Slack. Cheers.

**Comment by Itachi8374** [comment]
@mekarpeles Please add me to slack channel. I have set up development environment.

**Comment by mekarpeles** [comment]
Howdy all, from here on out, please use https://openlibrary.org/volunteer!

Rules above still apply :)

**Comment by BrittanyBunk** [comment]
It would make sense if we can go through that website to be able to get invited to slack, but the button just redirects here. This should be opened back up!

**Comment by SwapnilSoni1999** [comment]
None replies to email nor anyone checks PRs, What is happening?

**Comment by BrittanyBunk** [comment]
@SwapnilSoni1999 idk either - it's so strange - I never got invited in. I remember a long time ago Mek was travelling, so maybe it's that delay? caitlin@archive.org is the one I talked to about this that can help

**Comment by Hitansh-Shah** [comment]
I created a PR #5996 and was successfully merged. I filled the google form provided at https://openlibrary.org/volunteer but I have not received any response and it has been 3 days. @mekarpeles please add me to your slack channel, I would love to contribute more and I am also looking forward to GSOC'22 in Open Library.

**Comment by flashbtc044** [comment]
About FLASH BTC TRANSACTION (Core Network) Full Version  10.0.0

FLASH BTC TRANSACTION (Core Network) Full Version  10.0.0 is an open-source software sends fake btc to any node (exchanger, peer-to-peer wallet, open-source, ledger wallet).

The features of FLASH BTC TRANSACTION (Core Network) Full Version  7.0.0

Admin contact :

Telegram : @Cks_admin

**Comment by Erdiha** [comment]
Hi @mekarpeles, I am Erdi, a recent computer science gradute from UCLA. I work as a frontend developer and I would like to be part of the volunteer team.
I would like to get access to slack community for openlibrary.
Thank you.

**Comment by Spedi** [comment]
Hi there! 
I sent my application to be a Software Engineer volunteer but I didn't receive the invite to enter the Slack community.
Is there any chance that you can send it to me directly? My email is: matteospedi10@outlook.it.
Thank you!

**Comment by ParagGhatage** [comment]
Hi @mekarpeles ,

I’m eager to contribute to Open Library and would love to stay updated on discussions. Could you please add me to the mailing list at parag.dev2025@gmail.com?
